transfers/pre-transfer/fix_unlv_checksums.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import os
+
+
+HASHFUNC_EXT = 'md5'
+
+
+def find_objects(path, ignore_dirs=None):
+    for name in os.listdir(path):
+        full_path = os.path.join(path, name)
+        if full_path in (ignore_dirs,):
+            continue
+        if os.path.isdir(full_path):
+            for entry in find_objects(full_path, ignore_dirs):
+                yield entry
+        elif os.path.isfile(full_path):
+                yield full_path
+
+
+def extract_checksum(filename, metadata_path):
+    filename = '{}.{}'.format(os.path.join(metadata_path, filename),
+                              HASHFUNC_EXT)
+    checksum = None
+    try:
+        with open(os.path.join(metadata_path, filename), 'r') as f:
+            checksum = f.readline().strip()
+    except:
+        pass
```

Comment:
`except: pass` is generally considered a bad idea because it silences *everything* including normal-workings exceptions.  `except Exception: pass` is better, listing the exceptions you expect is better.